### ISSUE TITLE
Create rspc app

### ISSUE THREAD

**Issue by oscartbeaumont** [oscartbeaumont/rspc#54]
Framework:
 - [x] Tauri
 - [x] Axum
 - [ ] Actix-web (not supported by rspc yet)
 - [ ] Netlify functions (working in alpha rspc)
 - [ ] Cloudflare workers (working in alpha rspc)

Setup framework with rspc.

Database:
 - [x] Prisma Client Rust
 - [ ] Sqlx (Long term goal, not supported by rspc yet)
 - [ ] SeaORM (Long term goal, not supported by rspc yet)
 - [ ] None

Setup in rspc context.

Frontend:
 - [x] SolidJS Typescript
 - [x] React Typescript
 - [ ] Next.js (Long term goal)
 - [ ] None

Pass off the Vite for this setup process if possible.

Extras:
 - [ ] TailwindCSS
 - [ ] Tokio Tracing
 - [ ] Zer -> Unreleased auth library I am working on.

---

 - [ ] Automated system for testing every combination.
 - [ ] Randomly picking default port for the webserver when scaffolding project would be nice. Maybe 3005-3050 or something.

**Comment by oscartbeaumont** [comment]
Some Future work:
 - [ ] Dir should be allowed as first command line arg
 - [ ] Automatically running `cargo prisma generate` and `{node_package_manager} install`

I think @VarunPotti has started on these.

**Comment by oscartbeaumont** [comment]
- [ ] Setting Tauri & No UI framework doesn't work properly.

**Comment by VarunPotti** [comment]
> Some Future work:
> 
> * [ ]  Dir should be allowed as first command line arg
> * [ ]  Automatically running `cargo prisma generate` and `{node_package_manager} install`
> 
> I think @VarunPotti has started on these.

Yep will push soon and try to get the other issue fixed tomorrow after I come back from school.
